lambdas/service/app.py

```diff
@@ -1285,14 +1301,18 @@ def get_summary():
                                              authentication=request.authentication)
 
 
-def manifest_route(*, fetch: bool, initiate: bool):
+def manifest_route(*, fetch: bool, initiate: bool, curl: bool):
+    method = 'GET' if not initiate else 'POST' if curl else 'PUT'
```

Comment:
If there are invalid or nonsensical combinations of argument values, you should assert against them. It might be better to use tree of `if` statements rather than a tertiary expression.

I think `curl` might benefit from having a default of `False`, since `True` is a special case.